pom.xml

```diff
@@ -31,6 +31,16 @@
             <version>2.7.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>6.4.0.CR1</version>
```

Comment:
I would recommend to use final version of hibernate-core. check it here: [https://mvnrepository.com/artifact/org.hibernate/hibernate-core](url)